src/login/components/LoginApp/ResetPassword/CountDownTimer.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+let count = 0, counter = null, minute = "", second = "";
+
+const timer = () => {
+  count = setLocalStorage("count", count - 1);
```

Comment:
Please see in`src/login/components/GroupManagement/Groups/GroupsParent.js` how we have been asked to write code for use of local storage. This is to make the local storage variables searchable to prevent them from being used twice and break its intended functionality.

Reply:
I added variables for count and one for email, we need to keep "email" in local storage if page is reloaded to enable counter. 
what do you think about creating a file for all local storage variable? 
if you agree with that I will make a separate PR for that.